### ISSUE TITLE
MCinaBox Optifine

### ISSUE THREAD

**Issue by ThatGifIsNice** [AOF-Dev/MCinaBox#769]
Is it possible to have optifine without forge ?

**Comment by AyHa1810** [comment]
Yes, it is, but you can't use the optifine installer, so you have to install it manually. Get 1.12.2 optifine here for now #771

**Comment by ZeyMax** [comment]
Can you do optifine for 1.16.5 pls ?
